tests/playlists.py

```diff
@@ -776,6 +776,18 @@
 2.aac
 '''
 
+CUE_OUT_WITH_DURATION_KEY_PLAYLIST = '''#EXTM3U
+#EXT-X-TARGETDURATION:6
+#EXT-X-CUE-OUT:DURATION=11.52
```

Comment:
I think there is also a case for `#EXT-X-CUE-OUT:30.000` https://docs.aws.amazon.com/mediatailor/latest/ug/hls-ad-markers.html#hls-ad-markers-cue

Reply:
That case should be handled by existing code (`_cueout_simple`), assuming this PR doesn't change that.

Reply:
It is handled by the existing cue-out simple